boring/src/pkey.rs

```diff
@@ -281,6 +281,24 @@ where
         private_key_to_der,
         ffi::i2d_PrivateKey
     }
+
+    // This isn't actually PEM output, but `i2d_PKCS8PrivateKey_bio` is documented to be
+    // "identical to the corresponding PEM function", and it's declared in pem.h.
+    private_key_to_pem! {
+        /// Serializes the private key to a DER-encoded PKCS#8 PrivateKeyInfo structure.
+        ///
+        /// This corresponds to [`i2d_PKCS8PrivateKey_bio`].
+        ///
+        /// [`i2d_PKCS8PrivateKey_bio`]: https://www.openssl.org/docs/man1.0.2/man3/i2d_PKCS8PrivateKey_bio.html
```

Comment:
```suggestion
        /// [`i2d_PKCS8PrivateKey_bio`]: https://www.openssl.org/docs/man1.1.1/man3/i2d_PKCS8PrivateKey_bio.html
```
Super nitpick, but the docs are also a little better for 1.1.1